lib/markdown.ts

```diff
@@ -42,7 +42,7 @@ export function findSectionHeader(
   str: string
 ): string | undefined {
   // Get all the matching strings.
-  const regexp = new RegExp(`## .*${str}.*\n`, 'gi');
+  const regexp = new RegExp(`## .*${str}.*\n`, 'giu');
```

Comment:
this may be a breaking change

Reply:
That's a fair concern but realistically I don't think it's an issue and certainly not a blocker. This code is mostly just used for searching for sections like `## Rules`, `## Options`, `## Config` where the flag is unlikely to have an effect. Also I'm still open to breaking changes although I plan to release v1.0.0 very soon.